x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/DateUtils.java

```diff
@@ -216,6 +216,45 @@ public static TemporalAccessor atTimeZone(TemporalAccessor ta, ZoneId zoneId) {
         }
     }
 
+    public static String toRome(int month) {
+        switch (month) {
+            case 1:
+                return "I";
+            case 2:
+                return "II";
+            case 3:
+                return "III";
+            case 4:
+                return "IV";
+            case 5:
+                return "V";
+            case 6:
+                return "VI";
+            case 7:
+                return "VII";
+            case 8:
+                return "VIII";
+            case 9:
+                return "IX";
+            case 10:
+                return "X";
+            case 11:
+                return "XI";
+            case 12:
+                return "XII";
+            default:
+                return "";
```

Comment:
Your use is safe, but as a "util", I'd be more comfortable with an exception here.